Sources/ComposableArchitecture/SwiftUI/IdentifiedArray.swift

```diff
@@ -237,6 +237,20 @@ where Element: Identifiable, ID == Element.ID {
   public init() {
     self.init([], id: \.id)
   }
+
+  public mutating func replaceSubrange<C, R>(_ subrange: R, with newElements: C) where C : Collection, R : RangeExpression, Self.Element == C.Element, Self.Index == R.Bound {
```

Comment:
Looks like we're also getting an 11.3 failure on this line for using `Self`. Can we try to omit it and see if that builds?
```suggestion
  public mutating func replaceSubrange<C, R>(_ subrange: R, with newElements: C) where C : Collection, R : RangeExpression, Element == C.Element, Index == R.Bound {
```